css/common.css

```diff
@@ -2100,3 +2100,4 @@ input::-webkit-inner-spin-button {
     -webkit-appearance: none;
     margin: 0;
 }
+

```

Comment:
Sometimes in your pull requests, there are new lines appearing at the end of files like this one. In your PHPStorm, in Preferences > Editor > General, is the Ensure every saved file ends with a line break option checked?

Reply:
No, it is not checked. So there should not have a new line at the end?

Reply:
>No, it is not checked.

Keep the option not checked.

>So there should not have a new line at the end?

This file should not have a new line at the end. The new line at the end of the file means that it is listed as a changed file in this pull request. You can delete the new line and commit this change so that this file is not in the pull request anymore.